lib/locales/fr/frontend/organizations.json

```
@@ -1,7 +1,6 @@
 {
   "all_organizations_title": "Organisations lançant des Hackathons et Défis",
   "all_organizations_page_description": "Startups, Entreprises, Universités, Investisseurs, Communautés Tech, Écoles de Codage et d'autres Organisations organisent des hackathons sur la plateforme de hackathon TAIKAI",
-  "launch_your_challenge": "Lancez votre Hackathon",
```

We are using this on `components/products/launch-challenge-modal.js`
```suggestion
  "launch_your_challenge": "Lancez votre Hackathon",
```